app/src/main/java/com/dgaf/happyhour/Model/DrawerListAdapter.java

```diff
@@ -61,72 +87,62 @@ public View getView(int position, View convertView, ViewGroup parent) {
             case DAYS:
-                    inflater = (LayoutInflater) mContext.getSystemService(Context.LAYOUT_INFLATER_SERVICE);
-                    v = inflater.inflate(R.layout.weekday_item, null);
-
-                    Button monday =  (Button) v.findViewById(R.id.monday);
-                    Button tuesday = (Button) v.findViewById(R.id.tuesday);
-                    Button wednesday = (Button) v.findViewById(R.id.wednesday);
-                    Button thursday = (Button) v.findViewById(R.id.thursday);
-                    Button friday = (Button) v.findViewById(R.id.friday);
-                    Button saturday = (Button) v.findViewById(R.id.saturday);
-                    Button sunday =  (Button) v.findViewById(R.id.sunday);
-
-                    //TODO Respond to when the buttons are clicked
-                    monday.setOnClickListener(new View.OnClickListener() {
-                        public void onClick(View v) {
-                            Toast.makeText(mContext, "monday was clicked", Toast.LENGTH_SHORT).show();
-                        }
-                    });
-
-                    tuesday.setOnClickListener(new View.OnClickListener() {
-                        public void onClick(View v) {
-                            Toast.makeText(mContext, "tuesday was clicked", Toast.LENGTH_SHORT).show();
-                        }
-                    });
-
-                    wednesday.setOnClickListener(new View.OnClickListener() {
-                        public void onClick(View v) {
-                            Toast.makeText(mContext, "wednesday was clicked", Toast.LENGTH_SHORT).show();
-                        }
-                    });
-
-                    thursday.setOnClickListener(new View.OnClickListener() {
-                        public void onClick(View v) {
-                            Toast.makeText(mContext, "thursday was clicked", Toast.LENGTH_SHORT).show();
-                        }
-                    });
-
-                    friday.setOnClickListener(new View.OnClickListener() {
-                        public void onClick(View v) {
-                            Toast.makeText(mContext, "friday was clicked", Toast.LENGTH_SHORT).show();
-                        }
-                    });
-
-                    saturday.setOnClickListener(new View.OnClickListener() {
-                        public void onClick(View v) {
-                            Toast.makeText(mContext, "saturday was clicked", Toast.LENGTH_SHORT).show();
-                        }
-                    });
-
-                    sunday.setOnClickListener(new View.OnClickListener() {
-                        public void onClick(View v) {
-                            Toast.makeText(mContext, "sunday was clicked", Toast.LENGTH_SHORT).show();
-                        }
-                    });
+                inflater = (LayoutInflater) mContext.getSystemService(Context.LAYOUT_INFLATER_SERVICE);
```

Comment:
We might be able to put some of this code in a helper method since its pretty boilerplate setting up image views and listeners